e2e_correctness/test_modules.py

```diff
@@ -174,11 +187,49 @@ def _run_test(test_dir: str, memgraph_db: Memgraph, neo4j_driver: neo4j.BoltDriv
     ), "The graphs are not equal, check the logs for more details"
 
 
+def _run_path_test(
+    test_dir: str, memgraph_db: Memgraph, neo4j_driver: neo4j.BoltDriver
+):
+    """
+    Run input queries on Memgraph and Neo4j and compare path results after running test query
+    """
+    input_cyphers = (
+        test_dir.joinpath(TestConstants.INPUT_FILE).open("r").readlines()
+    )
+    mg_execute_cyphers(input_cyphers, memgraph_db)
+    logger.info(f"Imported data into Memgraph from {input_cyphers}")
+    neo4j_execute_cyphers(input_cyphers, neo4j_driver)
+    logger.info(f"Imported data into Neo4j from {input_cyphers}")
```

Comment:
You can ignore this comment if it doesn't make sense but I would print something along the line of:
logger: Importing from cypher: input_cyphers
logger: Imported data into memgraph
logger: Imported data into neo4j
Since you are printing the same possibly huge data twice